sip-core/src/main/java/de/ikor/sip/foundation/core/translate/logging/SIPPatternLayoutEncoder.java

```diff
@@ -1,16 +1,14 @@
 package de.ikor.sip.foundation.core.translate.logging;
 
-import ch.qos.logback.core.Layout;
+import ch.qos.logback.classic.spi.ILoggingEvent;
```

Comment:
I'm just going to put my comment on the leftover warnings here since it's tied to ILoggingEvent.
They have deprecated _getMdc()_ method:
![image](https://user-images.githubusercontent.com/17590248/168808292-11418a03-506f-4ad1-8c91-61934a80e544.png)
And replaced it with _e.getMDCPropertyMap();_

As far as I can see, Mapstruct actually maps both MDCPropertyMap and MDC but in the background this is the implementation:
```
public Map<String, String> getMdc() {
        return this.mdcPropertyMap;
    }
```

We can just add an ignore on the MDC filed for mapstruct. Take a look at:
https://github.com/mapstruct/mapstruct/issues/1151
and 
https://www.baeldung.com/mapstruct-ignore-unmapped-properties